config/routes.rb

```diff
@@ -7,11 +7,11 @@
   get '/api/v1/chat_service', to: 'api/v1/breakdowns#show'
   get 'api/v1/holidays', to: "api/v1/holidays#index"
 
-  get 'api/v1/users/:id/tasks', to: 'api/v1/tasks#index'
-  post 'api/v1/users/:id/tasks', to: 'api/v1/tasks#create'
-  get 'api/v1/users/:id/tasks/:id', to: 'api/v1/tasks#show'
-  patch 'api/v1/users/:id/tasks/:id', to: 'api/v1/tasks#update'
-  delete 'api/v1/users/:id/tasks/:id', to: 'api/v1/tasks#destroy'
+  get 'api/v1/users/:user_id/tasks', to: 'api/v1/tasks#index'
+  post 'api/v1/users/:user_id/tasks', to: 'api/v1/tasks#create'
+  get 'api/v1/users/:user_id/tasks/:id', to: 'api/v1/tasks#show'
+  patch 'api/v1/users/:user_id/tasks/:id', to: 'api/v1/tasks#update'
+  delete 'api/v1/users/:user_id/tasks/:id', to: 'api/v1/tasks#destroy'
```

Comment:
good move changing the dynamic part of the url to use a better name so you could grab it in the params without ambiguity